app/assets/javascripts/application.js

```diff
@@ -8,6 +8,9 @@
 //= require highlight
 //= require clipboard
 //= require form_errors
+//= require server_load_chart
```

Comment:
Would it make sense to call this file `charts.coffee`? That way we don't lock it down to just server load charts